src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, isMarried, partnerName, sex } = person;
+  let relationshipStatus;
+
+  if (isMarried) {
+    relationshipStatus = sex === 'm' ? 'wife' : 'husband';
+  } else {
+    relationshipStatus = 'not married';
+  }
```

Comment:
you can use ternary operator instead
```suggestion
  const relationshipStatus = sex === 'm' ? 'wife' : 'husband'
```